pyvips/enums.py

```diff
@@ -884,63 +893,6 @@ class FailOn(object):
     WARNING = 'warning'
 
 
-class ForeignPpmFormat(object):
```

Comment:
Was this class deleted by mistake?

Reply:
I've regenerated this file with commit b41f70728850e0cab027699806c9025b0e56e024 which seems to fix that.

Reply:
Huh strange, I regenerated the file too. Perhaps my pyvips install picked up the wrong libvips.

Thanks!